docs/consume-packages/includes/restore-nuget-exe-cli.md

```diff
@@ -1,13 +1,21 @@
-Use the [restore](../../reference/cli-reference/cli-ref-restore.md) command, which downloads and installs any packages missing from the *packages* folder.
+The NuGet CLI [restore](../../reference/cli-reference/cli-ref-restore.md) command downloads and installs any packages that are missing from project package directories. The command works on projects that use either `PackageReference` or *packages.config* for package references.
```

Comment:
Links to https://docs.microsoft.com/en-us/nuget/reference/packages-config and https://docs.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files may help here

Reply:
@JonDouglas- This include is used in the consume-packages article, so a link to consume-packages would be circular.